server/public/styles.css

```diff
@@ -115,3 +114,38 @@ h3 {
   text-align: center;
   padding: 70px;
 }
+
+.backbtn {
+  color: darkgray;
+}
+
+#back-icon {
+  color: darkgray;
+  padding-left: 7px;
+  font-size: 29px;
+  padding-top: 25px;
+}
+
+.eventdetail-container {
+  border: solid white;
+  margin: auto;
+  padding: 10px;
+  width: 60%;
+  margin-top: 35px;
+  box-shadow: 0 4px 8px 0 rgba(0, 0, 0, 0.2);
+  height: 81vh;
+}
+
+.event-infor {
+  text-align: center;
+}
+
+.edit-btn {
+  background-color: #007bff;
+  border: none;
+  color: white;
+  padding: 10px 32px;
+  text-decoration: none;
+  margin: 4px 2px;
+  cursor: pointer;
+}
```

Comment:
Make sure you consistently use `rem` instead of `px` for `font-size`s, `margin`s, `padding`s, `border`s and other properties with small sizes for consistency: https://lfz-lms.herokuapp.com/code-guides/Learning-Fuze/curriculum/effective-css